pairinteraction/CMakeLists.txt

```diff
@@ -31,6 +31,8 @@ if(MSVC)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /bigobj")
 endif()
 
+add_compile_options(-Wno-unused-local-typedefs)
```

Comment:
This currently breaks the MSVC build.  Please remove it and append it to this:
```cmake
set_source_files_properties(${pairinteraction_SRCS} PROPERTIES COMPILE_FLAGS "-Wall -Wextra -pedantic -Werror")
```

Reply:
done